catalog/edf-methanesat-ee/methane.md

```diff
@@ -0,0 +1,8 @@
+
+
+Methane is a potent greenhouse gas that has more than 80 times the warming power
+of carbon dioxide over the first 20 years after it reaches the atmosphere. At
+least 30% of today's global warming is driven by methane from human actions.
+Cutting methane emissions associated with human activities - including avoidable
+emissions from oil and gas operations, agriculture, and waste management - is
+the single fastest way to slow the rate of global warming.
```

Comment:
Could you link to something that clarifies what "the single fastest way" means specifically?

Reply:
I didn't write this and don't have a link handy, I can ask for one tomorrow.  In the meantime, though I did update the text with yours, to say "is one of the fastest ways to slow the rate of global warming"

Reply:
The current text still says "the single fastest way". This is fine for now, but let's rephrase and add a link (https://iopscience.iop.org/article/10.1088/1748-9326/ac4f10 ?) later.